src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/Configurations/UserTestCorrectAnswerConfiguration.cs

```diff
@@ -16,6 +17,8 @@ public void Configure(EntityTypeBuilder<UserTestCorrectAnswer> builder)
 
             builder.HasKey(ut => ut.Id);
 
+            builder.Property(ut => ut.AnsweringTimeOffset).HasConversion(new TimeSpanToTicksConverter());
```

Comment:
please rename it to `AnsweringTimeTicks`, because it is not offset anymore

Reply:
Only in the database we'll have ticks. EF will automatically convert ticks back to the timespan that will be still our offset from the test start date. I suggest sticking to that name.